nautobot/extras/tests/test_filters.py

```diff
@@ -1308,8 +1312,12 @@ def test_peer_id(self):
         params = {"peer_id": [self.devices[2].pk]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
 
+    @skip("Skipping RelationshipAssociation is a through model.")
+    def test_q_filter_exists(self):
+        """Test is not applicable for model."""
+
 
-class RelationshipModelFilterSetTestCase(FilterTestCases.FilterTestCase):
+class RelationshipAssociationFilterSetTestCase(FilterTestCases.FilterTestCase):
```

Comment:
good spot :)

Reply:
This is actually an almost identical copy of the previous test case. We should really combine the two.